hbw/app/services/hbw/with_definitions.rb

```diff
@@ -20,14 +20,29 @@ def with_cached_definition(task, entity_code_key, cache_key)
       zip_one(task, process_definitions, entity_code)
     end
 
-    def process_definitions
+    def process_definitions(user_email = nil)
```

Comment:
I think it should rather be two methods. Such arguments bring uncertainty.

Reply:
ok